math/matrix/inc/TVectorT.h

```diff
@@ -27,15 +27,15 @@
 template<class Element> class TVectorT : public TObject {
 
 protected:
-   Int_t    fNrows;                // number of rows
-   Int_t    fRowLwb;               // lower bound of the row index
-   Element *fElements;             //[fNrows] elements themselves
+   Int_t    fNrows{0};                // number of rows
```

Comment:
I agree, it seems reasonable to use the simple and readable copy-init for scalar types.